harpy/common/create_pixi.py

```
@@ -0,0 +1,117 @@
+def create_pixi_dockerfiles():
+    '''
+    Using the defined environments, create a series of folders where each has a dockerfile
+    and pixi.toml file to create one of the environments.
+    '''
+    shutil.rmtree("container", ignore_errors=True)
+    for env,deps in environ.items():
+        os.makedirs(f"container/{env}", exist_ok=True)
+        with open(f"container/{env}/Dockerfile", "w") as dockerfile:
+            dockerfile.write(dockerfile_text)
+        if env == "report":
+            subprocess.run(f"pixi init container/{env} -c conda-forge -c r".split())
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)